Gemfile.lock

```diff
@@ -365,6 +363,7 @@ GEM
     zeitwerk (2.5.4)
 
 PLATFORMS
+  ruby
```

Comment:
Why do we need this now?

Reply:
this shouldn't be there, removed it 3 times already and added it back again :disappointed_relieved: